docs/ops.html

```diff
@@ -1576,12 +1576,17 @@ <h4 class="anchor-heading"><a id="remote_jmx" class="anchor-link"></a><a href="#
       <tr>
         <td>Partition counts</td>
         <td>kafka.server:type=ReplicaManager,name=PartitionCount</td>
-        <td>mostly even across brokers</td>
+        <td>The number of partitions assigned to the broker.</td>
```

Comment:
The 3rd column is the normal value, not a description, so "mostly even across brokers" is actually correct.